src/features/classification/AddClassificationBadge.js

```diff
@@ -2,14 +2,14 @@
 import * as React from 'react';
 import { FormattedMessage } from 'react-intl';
 
-import IconAddTags from '../../icons/general/IconAddTags';
+import IconSecurityClassification from '../../icons/general/IconSecurityClassification';
 import { bdlGray62 } from '../../styles/variables';
 import messages from './messages';
 import './AddClassificationBadge.scss';
 
 const AddClassificationBadge = () => (
     <h1 className="bdl-AddClassificationBadge">
-        <IconAddTags color={bdlGray62} height={10} width={10} strokeWidth={3} />
+        <IconSecurityClassification color={bdlGray62} height={13} width={11} strokeWidth={1} />
```

Comment:
The values for `height`, `width`, and `strokeWidth` match the default values, so you can remove the attributes here.